### ISSUE TITLE
op.io.OpenpnmIO.load_object_from_dict creates but does not return the project

### ISSUE THREAD

**Issue by 1minus1** [PMEAL/OpenPNM#1498]
This function is missing the line:

`return project`

It returns None.

**Comment by ma-sadeghi** [comment]
You're right, I think it should ideally return the `project` object. But, you can retrieve the loaded project by looking into the `Workspace` object via:
```python
ws = op.Workspace()
print(ws)
```
```bash
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
OpenPNM Version 2.3.1 Workspace
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 sim_01
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 Object Name     Object ID                                                        
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 net_01          <openpnm.network.GenericNetwork object at 0x24140fb4fc0>         
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 sim_02
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 Object Name     Object ID                                                        
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 net_01          <openpnm.network.GenericNetwork object at 0x241406751a8>         
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 sim_03
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 Object Name     Object ID                                                        
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 net_01          <openpnm.network.GenericNetwork object at 0x24140fb43b8>         
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
```
```python
print(ws.keys())
```
```
dict_keys(['sim_01', 'sim_02', 'sim_03'])
```
```
prj_1 = ws["sim_01"]
```

**Comment by jgostick** [comment]
Fixed in PR #1542
